lib/content-services/src/lib/permission-manager/components/user-name-column/user-name-column.component.ts

```diff
@@ -72,7 +72,7 @@ export class UserNameColumnComponent implements OnInit {
 
     private updateGroup(group: Group) {
         if (group) {
-            this.displayText$.next(group.displayName);
+            group.displayName ? this.displayText$.next(group.displayName) : this.displayText$.next(group.id);
```

Comment:
Small thing but probably can be just: 

this.displayText$.next(group.displayName || group.id);

Reply:
right, ty